### PR TITLE
[JP 2020] Add DevOpsDays João Pessoa 2020

### PR DESCRIPTION
**Pull request by MarceloFreire88**
Add the DevOpsDays  João Pessoa 2020.

The e-mail with the organizers list has already been sent.

### CODE REVIEW

File: data/events/2020-joao-pessoa.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Daniela Pitta"
+    role: "Organizer"
+    bio: ""
+    linkedin: "http://linkedin.com/in/dlpitta"
+    website: "https://www.dlpitta.me/"
+    github: "dlpitta"
+    twitter: "@dlpitta"
+    image: "daniela.jpg"
```
Comment (somatorio) [inline data/events/2020-joao-pessoa.yml]: She's not at the list in the email you mentioned

Comment (MarceloFreire88) [reply data/events/2020-joao-pessoa.yml]: Ssorry, i forgot. I sent a new email adding the new informations about Daniela.

**Comment by somatorio** [comment]
waiting for someone with access to sync the email and slack with the list, then we can merge it :)

**Comment by MikeRosTX** [comment]
> waiting for someone with access to sync the email and slack with the list, then we can merge it :)

Working on it now @somatorio. 👍

**Pull request merged**